helper-cli/src/main/kotlin/commands/SetDependencyRepresentationCommand.kt

```diff
@@ -0,0 +1,176 @@
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.ProgramResult
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.enum
+import com.github.ajalt.clikt.parameters.types.file
+
+import org.ossreviewtoolkit.analyzer.AnalyzerResultBuilder
+import org.ossreviewtoolkit.model.AnalyzerResult
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.model.ProjectAnalyzerResult
+import org.ossreviewtoolkit.model.readValue
+import org.ossreviewtoolkit.model.utils.DependencyGraphConverter
+import org.ossreviewtoolkit.model.writeValue
+import org.ossreviewtoolkit.utils.expandTilde
+
+/**
+ * A command to convert the dependency representation in an ORT result file between the different supported formats.
+ *
+ * The input file can contain the serialized form of either a full [OrtResult], an [AnalyzerResult], or a single
+ * [ProjectAnalyzerResult].
+ */
+class SetDependencyRepresentationCommand : CliktCommand(
+    help = "Convert an ORT result from the dependency tree representation to the dependency graph format."
+) {
+    /**
+     * An enumeration class representing the different supported formats to represent dependencies in an ORT result.
+     * It is used to define the target format that should be set by this command.
+     */
+    enum class TargetFormat {
+        /** The dependency graph format. */
+        GRAPH {
+            override fun convert(result: AnalyzerResult): AnalyzerResult =
+                DependencyGraphConverter.convert(result)
+        };
+
+        /**
+         * Convert the given [result] to the target format defined by this literal.
+         */
+        abstract fun convert(result: AnalyzerResult): AnalyzerResult
+    }
+
+    private val ortFile by option(
+        "--ort-file", "-i",
+        help = "The ORT result file to read as input. This can be the serialized form of an OrtResult, an " +
+                "AnalyzerResult, or a ProjectAnalyzerResult."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    private val outputFile by option(
+        "--output-file", "-o",
+        help = "The file in which to write the result of the conversion. If missing, the input file is overwritten."
+    ).convert { it.expandTilde() }
+        .file(mustExist = false, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+
+    private val targetFormat by option(
+        "--target", "-t",
```

Comment:
Why not `"--format", "-f"`, as "target" could be misunderstood as the output file?